mgmtsystem_action/models/mgmtsystem_action_stage.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields
+
+
+class MgmtsystemActionStage(models.Model):
+    """This object is used to defined different stage for actions."""
+
+    _name = 'mgmtsystem.action.stage'
+    _description = "Action stages"
+    _order = 'sequence'
+
+    name = fields.Char('Stage Name', required=True, translate=True)
+    sequence = fields.Integer(
+        'Sequence', help="Used to order stages. Lower is better.", default=1)
```

Comment:
I suggest using a larger default, such as 100.
Then, be default new Stages will be at the end and not before the existing ones.